clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeleteExecutor.scala

```diff
@@ -103,10 +103,14 @@ class DeleteExecutor[TI <: TransferIndex](deleteCommand: DeleteCommand[TI])
   )(implicit ec: ExecutionContext): Future[UpsertId] = {
     val pathsToDelete = existingMetadata.pathsToDelete.filter { path =>
       val pathExists = ioUtil.googleObjectExists(path)
+      val err = s"'$path' associated with $prettyKey does not exist in the cloud."
       if (!pathExists) {
-        logger.warn(
-          s"'$path' associated with $prettyKey does not exist in the cloud."
-        )
+        if (!deleteCommand.forceDelete) {
+          throw new IllegalStateException(
+            err + " Use --force-delete to delete this file."
```

Comment:
Oops, missed it on first pass. `to delete this file` is confusing because the 1st half of the message says the file doesn't exist. `to mark the record as deleted in Clio` is more accurate.